### DISCUSSION TITLE
1016 회의 안건

### DISCUSSION THREAD

**Discussion by kr-atst** [NoR3ST/nor3st-backend#19] [Announcements]
1. 프론트 수정 보완 
2. A.I 모델 리뷰 및 브리핑
3. 교재방안
4. 서비스 기획안 디렉션 회의

**Comment by jeanDeluge** [comment]
1. 발음, 억양, 강세는 AI 에서 점수 채점을 연구해야 함
2. 파형 보고 비교할 수 있는 그래프는 프론트엔드에서 해야함 > @youyeul301 님 확인부탁드려요
3. 서비스 기획안
 - 관리자 대시보드 내용이 많아서 축소필요
 - 진도 관리는 사람마다 다른 경우 어떻게 해야할 것인가
